tools/submission/submission_checker.py

```diff
@@ -1049,6 +1049,184 @@
             "gpt3-99.9": {"SingleStream": 1024, "Server": 270336, "Offline": 1},
         },
     },
+    "v4.0": {
+        "models": [
+            "resnet",
+            "retinanet",
+            "rnnt",
+            "bert-99",
+            "bert-99.9",
+            "dlrm-v2-99",
+            "dlrm-v2-99.9",
+            "3d-unet-99",
+            "3d-unet-99.9",
+            "gptj-99",
+            "gptj-99.9",
+            "llama-v2-70b-99",
```

Comment:
I believe the model name should be `llama2-70b` according to the official website: https://huggingface.co/docs/transformers/main/model_doc/llama2

Reply:
Same for all occurrences

Reply:
I copied it from here: https://github.com/pgmpablo157321/inference/blob/f7c755a0e156f51c1684f984889b309b8afaffc2/language/llama2-70b/mlperf.conf#L59C1-L59C1

Should we change all the references in that folder as well?

Reply:
I think we should change all to llama2. The `llama-v2` name will cause confusion.